README.md

```diff
@@ -5,4 +5,18 @@ eZPlatform.com - eZ Systems' Developer Hub for the Open Source PHP CMS eZ Platfo
 
 This repo contains the source files for the ezplatform.com website, built on eZ Platform and hosted by the eZ Systems team.
 
+![ezplatform.com](https://raw.githubusercontent.com/ezsystems/ezplatform-com/master/web/assets/doc/images/ezplatform.jpg)
```

Comment:
True, but you could also upload image on GH and then only link to it. We did something like this on ezplatformo-demo (https://github.com/ezsystems/ezplatform-demo/blob/master/README.md). Clever, huh? 😈